appinventor/appengine/src/com/google/appinventor/server/project/youngandroid/YoungAndroidProjectService.java

```diff
@@ -697,16 +703,16 @@ public RpcResult build(User user, long projectId, String nonce, String target,
       // Keep in mind that large projects can lead to large APK files which
       // may not be loadable into many memory restricted devices, so we
       // may not want to encourage large projects...
-      if (zipFile.getContent().length > 10*1024*1024) { // 10 Megabyte size limit...
-        int zipFileLength = zipFile.getContent().length;
-        String lengthMbs = format((zipFileLength * 1.0)/(1024*1024));
+      if (zipFile.getContent().length > MAX_PROJECT_SIZE.get() * MB) {
+        double size = (double) zipFile.getContent().length / MB;
         RuntimeException exception = new RuntimeException(
-            "Sorry, can't package projects larger than 10Mb."
-            + " Yours is " + lengthMbs + "MB.");
+            String.format(Locale.getDefault(), ERROR_LARGE_PROJECT, MAX_PROJECT_SIZE.get(), size)
+        );
         CrashReport.createAndLogError(LOG, null,
```

Comment:
Perhaps we should get rid of the call to CrashReport, as this isn't really a crash. Calling crash report results in a SEVERE error being recorded in the log along with the stack trace.

Look at line 774. Probably want to get rid of the reference to larger then 5Mb (in the IOException handler).

Reply:
Done